src/pages/2019/12/year-in-review/index.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { Helmet } from 'react-helmet'
+import s from 'styled-components'
+
+import NavBar from '../../../../components/2019/year-in-review/Nav'
+import MultiMedia from '../../../../components/2019/year-in-review/Multimedia'
+import HeaderSection from '../../../../components/2019/year-in-review/HeaderSection'
+import Opinion from '../../../../components/2019/year-in-review/Opinion'
+import Street from '../../../../components/2019/year-in-review/Street'
+import News from '../../../../components/2019/year-in-review/News'
+import Sports from '../../../../components/2019/year-in-review/Sports'
+import { OPEN_SANS_REGULAR } from '../../../../utils/font'
+
+const Wrapper = s.div`
+    a:hover {
+        text-decoration: none;
+    }
+`
+
+const Footer = s.div`
+    ${OPEN_SANS_REGULAR}
+    text-align: center;
+    padding-top: 50px;
+    padding-bottom: 50px;
```

Comment:
there might be a padding-x or y or something like that that lets you do this in one line